tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3126,7 +3132,7 @@ <h2>Create a distance feature</h2>
 
       makeAsyncRequest_: function(url, thenDoFn, postData){
         var self = this;
```

Comment:
Using arrow function, now, you don't need this `self`

Reply:
done